spaces/cofix/index.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "CoFiX",
+  "network": "1",
+  "symbol": "COFI",
+  "skin": "cofi",
```

Skin field value should be similar to file name.
```suggestion
  "skin": "cofix",
```

---

fixed.